litmus-portal/graphql-server/graph/schema.graphqls

```diff
@@ -173,7 +197,12 @@ type Query{
 
   users: [User!]! @authorized
 
+  # [Deprecated soon]
   getScheduledWorkflows(project_id: String!): [ScheduledWorkflows]! @authorized
+
+  ListWorkflow(project_id: String!, workflow_ids: [ID] ): [Workflow]! @authorized
+
+  GetWorkflow(workflow_id: String!): Workflow @authorized
```

Comment:
We don't need this anymore right? ListWorkflows should handle this use-case?

Reply:
Yes, but, In ListWorkflow we have to send `project_id`. GetWorkflow is like Get operation for Workflows

Reply:
We actually need to keep the project_id in this one too, it is required for authorization. We can just verify if the user is part of the project and then proceed with getting the workflow.

Reply:
Okay, then this can be removed.